src/app/clusters/mode-base-server/mode-base-server.cpp

```diff
@@ -23,7 +23,6 @@
 #include <app/clusters/on-off-server/on-off-server.h>
 #include <app/reporting/reporting.h>
 #include <app/util/attribute-storage.h>
-#include <app/util/config.h>
```

Comment:
Why is config.h being removed?  That's what pulls in the definition of `EMBER_AF_PLUGIN_ON_OFF_SERVER`.....